apache2/mod_security2.c

```diff
@@ -833,7 +833,7 @@ static int hook_post_config(apr_pool_t *mp, apr_pool_t *mp_log, apr_pool_t *mp_t
  * Initialisation performed for every new child process.
  */
 static void hook_child_init(apr_pool_t *mp, server_rec *s) {
-    modsecurity_child_init(modsecurity);
+    modsecurity_child_init(mp, modsecurity);
```

Comment:
Do we need to change the ModSec API?

Reply:
since it's a fix and we do need to allocate for the struct, i think it's needed